README.md

```diff
@@ -1,2 +1,47 @@
-# Microverse-leaderboard
-Leaderboard list app, sending and receiving data from/to the Leaderboard API.
+# Leaderboard
+
+> Leaderboard list app, sending and receiving data from/to the Leaderboard API.
+
+## Built With
+- HTML
+- CSS
+-JAVASCRIPT
+-Webpack
+​
+## Live Demo (if available)
+
+- Live Demo: [@Live demo link](https://alphantihinduka.github.io/Microverse-leaderboard/dist/);
+
+## Getting Started
+
+#To get a local copy up and running follow these simple steps:
+
+-Open terminal and navigate to the folder you would like to put these files.
+-Type these commands into the terminal:
+-git clone git@github.com:AlphaNtihinduka/To-Do-List.git
+-cd todo
+-npm install
+-To build the project with webpack
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.